lib/parse.js

```diff
@@ -23,9 +23,7 @@ export function parseWithoutProcessing(input, options) {
     return new Helpers.SourceLocation(options && options.srcName, locInfo);
   };
 
-  let ast = parser.parse(input);
```

Comment:
Note: this only creates a reference to an object, so there is no memory allocation happening.

So if you want to keep it that way, please say so and I'll create a new PR with only the typo fixes.